streamServiceViewerStyles.css

```diff
@@ -186,6 +186,16 @@ section:first-of-type .config-header {
 	width: 100%;
 }
 
+.input-row {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.prev-element {
+	margin-right: 5px;
```

Comment:
Do we need to adjust the spaces/tabs of this line?

Reply:
Yes, I will run the formatter on this file